cache/fast_lru_cache.h

```diff
@@ -11,13 +11,22 @@
 #include <memory>
 #include <string>
 
+#include "cache/cache_key.h"
 #include "cache/sharded_cache.h"
 #include "port/lang.h"
 #include "port/malloc.h"
 #include "port/port.h"
 #include "rocksdb/secondary_cache.h"
 #include "util/autovector.h"
 
+constexpr uint8_t kCacheKeySize =
+    static_cast<uint8_t>(sizeof(ROCKSDB_NAMESPACE::CacheKey));
+
+constexpr double kLoadFactor = 0.7;
+
+constexpr uint32_t kProbingSeed1 = 0xbc9f1d34;
```

Comment:
How are these seeds obtained? How to update them if needed?
Please add a comment.

Reply:
These were just arbitrary seeds.